duckduckgo/iOS
# Send failed compilations pixel if needed

Task/Issue URL: https://app.asana.com/0/1203790657351911/1205404596674276/f

Description:
Introduce daily pixel containing information about failed content blocking rules compilations.

Steps to test this PR:

1. Run the app and make sure the pixel is not being sent.
1. Make compilation fail, by for instance adding some incorrect characters to temporary unprotected sites.
1. Notice the counter increases for a given step of compilation.
1. See that the pixel is being sent on the next app run and that statistics are being reset.
